app/js/importExport.js

```diff
@@ -25,7 +25,7 @@ const importData = (
       const fileReader = new FileReader();
       fileReader.onload = () => {
         try {
-          const json = JSON.parse(fileReader.result);
+          const json = JSON.parse(String(fileReader.result));
```

Comment:
I added this explicit cast to string because Flow says:
```
app/js/importExport.js:28
 28:           const json = JSON.parse(fileReader.result);
                                       ^^^^^^^^^^^^^^^^^ ArrayBuffer. This type is incompatible with the expected param type of
550:     static parse(text: string, reviver?: (key: any, value: any) => any): any;
                            ^^^^^^ string. See lib: /tmp/flow/flowlib_34e60cd5/core.js:550
```